src/AppInstallerCLI.sln

```diff
@@ -63,311 +70,521 @@ Global
 		binver\binver.vcxitems*{fb313532-38b0-4676-9303-ab200aa13576}*SharedItemsImports = 4
 	EndGlobalSection
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|Any CPU = Debug|Any CPU

```

Comment:
>Debug|Any CPU = Debug|Any CPU [](start = 2, length = 29)

The only thing that remains is that I don't think we need to have either Any CPU or Win32 architectures built here.  We should be able to use the existing ones.